packages/terra-clinical-item-view/src/ItemView.module.scss

```diff
@@ -113,7 +113,7 @@
   }
 
   .content-secondary-color {
-    color: var(--terra-clinical-item-view-content-secondary-color, #64696c);
+    color: var(--terra-clinical-item-view-content-secondary-color, rgba(28, 31, 33, 0.65));
```

Comment:
Since this will resolve #673 could you revert the change for disabling the color-contrast in this test? https://github.com/cerner/terra-clinical/pull/674/files#diff-9ee238e41af23d6170088980ae1de2f5R34-R35 ?

Reply:
Reverted 87f5525